1_fetch/src/fetch_wqp_data.R

```diff
@@ -3,23 +3,25 @@
 #' @description This function returns a data frame with a single column used
 #' to group the sites into reasonably sized chunks for downloading data.
 #' 
-#' @param siteids vector of character strings containing site identifiers
-#' @param max_sites_allowed integer indicating the maximum number
-#' of sites allowed in each download group. Defaults to 500.
+#' @param siteids_df data frame containing the site identifiers and total number of
+#' records available for each site. Must contain column `MonitoringLocationIdentifier`.
+#' @param max_sites_allowed integer indicating the maximum number of sites allowed
+#' in each download group. Defaults to 500.
 #' 
-#' @return returns a data frame with columns site id, site number, and an 
-#' additional column called `download_grp` which is made up of unique groups 
-#' to `group_by()` and then `tar_group()` for downloading.
+#' @return returns a data frame with columns site id, the total number of records,
```

Comment:
```suggestion
#' @return returns a data frame with columns site id, the total number of records (retains the column from `siteids_df`),
```

Reply:
I incorporated this suggestion when I was editing the function documentation to reflect the updated naming convention.